infrastructure/kube/keep-test/electrs-statefulset.yaml

```diff
@@ -0,0 +1,90 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
I thought about whether to use a `Deployment` or `StatefulSet` and came to the conclusion we could use any of them in our case (we need just one instance of a program with access to persistent data storage).

I chose `StatefulSet` because [most sources claim](https://stackoverflow.com/a/41707491) it is more appropriate for applications that stores some data (have state).

If I understand it correctly, the difference is that if we increased the number of replicas to more than `1`, the replicas would share the same volume in case of `Deployment`(which could corrupt data if two `electrs` instances would be writing to the same data) and would have separate volumes in case of `StatefulSet`. 

Do you think we should use `Delployment`? In our case (we need `1` replica) it should not matter.